Catalog.lua

```diff
@@ -1,14 +1,8 @@
--- require('CeresStdLib.Catalog')
-
 require('CeresStdLib.base.Basics')
-require('CeresStdLib.base.Native')
```

Comment:
The reason for Catalog to have *all* the modules is for user to easily pick them to c&p, so please return them back here.

Reply:
Looks like I overlooked that one.